hhs_facilities/tests/test_pull.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for running the geo conversion functions."""
+
+from unittest.mock import patch
+
+import pytest
+import pandas as pd
+import numpy as np
+
+from delphi_hhs_facilities.pull import _pull_data_iteratively, pull_data
+
+
+class TestPull:
+
+    @patch("delphi_hhs_facilities.pull.Epidata.covid_hosp_facility")
+    @patch("delphi_hhs_facilities.pull.Epidata.covid_hosp_facility_lookup")
+    def test__pull_data_iteratively(self, covid_hosp_facility_lookup, covid_hosp_facility):
```

Comment:
little nit, but this test uses `test__name` while the one below uses `test_name`. i'm not sure what the convention is in pytest, but i recommend naming them the same way.

Reply:
Updated

Reply:
doh, i see it was because the function name itself started with an underscore. either way is fine imo. looks good